### PR TITLE
Updates README.md and folder names

### PR DESCRIPTION
**Pull request by iRahulP**
Updated README with missing links of API contracts and restructured the titles in cronological fashion as per current structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,12 @@
 
 ## APIs
 
+- [Contributions](/contributions)
+- [Crypto Site](/crypto-site)
```
Comment (swarajpure) [inline README.md:6]: Can we just call it `Crypto`? You'll have to change the folder name and also the link to the folder -- what you see inside () brackets here

File: README.md [hunk 2]
```diff
@@ -2,9 +2,12 @@
...
 - [Login](/login)
-- [Users](/users)
 - [Members](/members)
+- [Notifications](/notifications)
 - [Pull Requests](/pull-requests)
+- [Roadmap Site](/roadmap-site)
```
Comment (swarajpure) [inline README.md:11]: We were earlier calling it roadmap site, but not anymore
Can we please have the name as `Challenges`? Again, you'll have to change the folder name and the link

Also, please make it alphabetical so it becomes easier to read 😉

**Review by swarajpure — CHANGES_REQUESTED** [review]
Congratulation on your first pull request in Real Dev Squad! 🥳 

Suggesting some changes. Please check.

Thank you! 😁

File: README.md (hunk 2 above)
Comment (iRahulP) [reply README.md:11]: I will update the changes accordingly! Thanks for the Review Swaraj.

**Review by swarajpure — APPROVED** [review]
Great. Thank you! 😄 
Congratulations on your pull request in Real Dev Squad! 🎉 🥳

**Pull request merged**